mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/positionTooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+interface TooltipProps {
+  /**
+  * Tooltip's children (ReactNode)
+  */
+  children: ReactNode;
+  /**
+  * Tooltip's content (string | ReactNode)
+  */
+  content: string | ReactNode;
+  /**
+  * Tooltip's class-style (string)
+  */
+  className: string;
```

Comment:
what if I don't want to use custom styles for tooltip? Why is it required?

Reply:
Done. Default className - tooltipDefault. 
I did lost class tooltipDefault empty, because I don't see markup.